internal/broker/testdata/TestIsAuthenticated/golden/successfully_authenticate_user_with_password/first_call

```diff
@@ -6,6 +6,6 @@ data: |-
     		"gecos": "saved-user",
     		"dir": "/home/test-user@email.com",
     		"shell": "/usr/bin/bash",
-    		"groups": [{"name": "saved-remote-group", "gid": "12345"}, {"name": "saved-local-group", "gid": ""}]
+    		"groups": [ {"name": "remote-group", "ugid": "12345"}, {"name": "linux-local-group", "ugid": ""} ]
```

Comment:
This is due to removing the `token.Valid()` check that we had before. I did that to go for something like "ok, if the access token is still going, we don't need to requery the provider for anything", but after our discussions regarding token and provider interactions, I thought it would be best to remove it.

I don't have a strong opinion on this, so if you want to keep it, I'm fine as well.

Reply:
No, that’s fine, I didn’t want that to be an unknown side-effect :)